js/app/historyStore.js

```diff
@@ -167,6 +174,21 @@ const HistoryStore = new Lang.Class({
         return item || null;
     },
 
+    get current_subset () {
+        if (this._current_subset)
+            return this._current_subset;
+        return null;
+    },
+
+    set_current_subset: function (model) {
```

Comment:
Again sorry to nitpick, but this one should be in the section marked "Common helper functions for history stores, not for use from other modules"